.evergreen/config.yml

```diff
@@ -1877,19 +1877,6 @@ tasks:
       - func: install dependencies
       - func: bootstrap mongo-orchestration
       - func: check resource management feature integration
-  - name: check-types-typescript-next
```

Comment:
Keep this task but add `disable: true` to it and a comment explaining when we should re-enable it?